core/src/components/menu/menu.tsx

```diff
@@ -705,9 +704,18 @@ export class Menu implements ComponentInterface, MenuI {
       this.forceClosing();
     }
 
-    if (!this.disabled) {
-      menuController._setActiveMenu(this);
+    if (document.contains(this.el)) {
```

Comment:
Can we use https://github.com/ionic-team/ionic-framework/blob/aa9ea60357c369452bc121c0d663be7dbf3da28e/core/src/utils/browser/index.ts#L25 instead?

Reply:
Addressed here: https://github.com/ionic-team/ionic-framework/pull/27814/commits/5437b79fadabffecbc57ffc875ec0269abb0bb7d